src/sentry/templatetags/sentry_react.py

```diff
@@ -95,10 +95,15 @@ def get_react_config(context):
         messages = []
         is_superuser = False
 
+    enabled_features = []
+
     if user:
         user = extract_lazy_object(user)
+    if not user.is_anonymous():
+        enabled_features = [project.organization.slug + ':' + project.slug + ':similarity-view'
+            for project in Project.objects.filter(organization=User.objects.get(id=user.id).get_orgs()) if
+            features.has('projects:similarity-view', project=project)]
```

Comment:
No, this is actually an O(n) of queries. I think there is a helper, but I'd have to look into it.